packages/atti-components/src/components/Slider/styles/SliderIndicator.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import Label from "../../Typographies/Label";
+import PropTypes from "prop-types";
+import SliderStep from "./SliderStep";
+
+const SliderIndicator = styled.span`
+  ${({leftPosition, moving, theme: {components: {slider: {sliderIndicator}}}}) => `
+    background-color: ${sliderIndicator.colors.backgroundColor};
+    border-radius: ${sliderIndicator.borders.radius};
+    height: ${sliderIndicator.sizes.height};
+    width: ${sliderIndicator.sizes.width};
+    left: ${`calc(${leftPosition}px - ${sliderIndicator.sizes.width.slice(0, -2) / 2}px)`};
+    
+    ${Label}{
+      display: ${moving ? 'block' : 'none'};
+      right: ${`calc(${sliderIndicator.sizes.width.slice(0, -2) / 2}px - ${sliderIndicator.valueLabel.width.slice(0, -2) / 2}px)`};
+    }      
+    
+    ${moving ? `box-shadow: ${sliderIndicator.shadows.hover};` :
+      `&:hover{
+        box-shadow: ${sliderIndicator.shadows.moving};
+      }`
+    }
+  `}
+  transition: 200ms box-shadow ease-out;
+  cursor: pointer;
+  position: relative;
+  user-select: none;
+  z-index: 2000;
```

Comment:
We should create an elevations property in the theme and use it as a reference for our z-index

Reply:
Done